### DISCUSSION TITLE
Hide method signature next to a statement

### DISCUSSION THREAD

**Discussion by IArnaut2** [scalacenter/scastie#882] [Q&A]
How do I hide method signatures, such as "(): scala.Unit", that appear next to statements? I always confuse them of being part of code.

![image](https://github.com/scalacenter/scastie/assets/57326929/b48b261b-8fe9-4010-9b05-c09f920f2ad7)

**Comment by IArnaut2** [comment]
Nvm, figured out.

**Comment by IArnaut2** [comment]
Turning off _Worksheet_ causes illegal starts of definition. Is there a way to keep _Worksheet_ on without signatures appearing?

**Comment by rochala** [comment]
Hey!
The thing that you see is a decoration. It shows value and type of each statement.
This is part of a worksheet and can't be disabled.

The worksheet basically is a super simple notebook, which automatically wraps your code into
```scala
object extends App {
}
```
so you don't need to write it, and can instantly start hacking.
This is why it causes illegal starts of the definition when you disable it, as this code is incorrect.
If you're using Scala 3 you should start the code by wrapping it in
```scala
@main def hello = 
  println("hello world")
```
